packages/popover/src/vaadin-popover.js

```diff
@@ -149,6 +149,18 @@ class PopoverOpenedStateController {
   }
 }
 
+/**
+ * Returns true if the popover overlay is the last one in the opened overlays stack, ignoring tooltips.
+ * @param {HTMLElement} overlay
+ * @return {boolean}
+ * @protected
+ */
+export const isLastOverlay = (overlay) => {
```

Comment:
Not sure if we need to export this, maybe can keep it as private function.

Reply:
Sorry, that was just a copy+paste. Removed.